tests/decryption/test_decrypt.py

```diff
@@ -145,3 +147,32 @@ def test_permission_error(self, tmp_path):
         output_dir.chmod(0o400)
         with pytest.raises(PermissionError):
             move_files(file_paths=[INPUT_DIR/"hello.txt"], output_dir=output_dir)
+
+
+class TestRemoveFiles:
+    """Test remove_files."""
+
+    def test_remove_files(self, files, tmp_path):
+        """Test that the files in a directory are removed successfully."""
+        assert all(file.exists() for file in files)
+        remove_files(tmp_path)
+        assert not any(file.exists() for file in files)
+
+    def test_empty_dir(self, tmp_path):
+        """Test that no error is raised when an empty directory is passed."""
+        empty_dir = tmp_path/"empty"
+        empty_dir.mkdir()
+        remove_files(empty_dir)
+        assert not any(empty_dir.iterdir())
```

Comment:
Maybe make this a bit more explicit?
```suggestion
        assert len(list(empty_dir.iterdir())) == 0
```
(assuming the two are equivalent, of course)